i3pystatus/openvpn/systemd.py

```diff
@@ -1,12 +1,12 @@
-from i3pystatus import IntervalModule
+from i3pystatus.openvpn import Backend
```

Comment:
Yup, that was a copy/paste error during development. This should be addressed now.